Runtime/Scripts/BaseMeshSync.Textures.cs

```diff
@@ -0,0 +1,258 @@
+            else {
+                mat.shader = shader;
+            }
+
+            bool usingOverride = false;
+
+            // If the given shader name did not exist, try to set up the material to be close to the given shader name:
+            if (!shaderExists) {
+                if (shaderName.ToLower() == "glass") {
+                    SetupGlassShader(mat);
+                    usingOverride = true;
+                }
+            }
+
+            // If the material was set up by meshsync but doesn't need that anymore, revert it to a standard material:
+            if (!usingOverride && Array.IndexOf(mat.shaderKeywords, MeshSyncConstants.MESHSYNC_OVERRIDE) >= 0) {
+                mat.CopyPropertiesFromMaterial(new Material(shader));
+            }
+        }
+
+        /// <summary>
+        /// Sets the material up to represent a glass shader from DCC tools.
+        /// </summary>
+        /// <param name="mat">The material to change.</param>
+        private static void SetupGlassShader(Material mat) {
+            mat.EnableKeyword(MeshSyncConstants.MESHSYNC_OVERRIDE);
+
+            mat.EnableKeyword(MeshSyncConstants._SURFACE_TYPE_TRANSPARENT);
+            mat.SetOverrideTag(MeshSyncConstants.RenderType, MeshSyncConstants.Transparent);
+            mat.SetFloat(MeshSyncConstants._Surface, 1);
+
+#if AT_USE_HDRP
+            SetupGlass_HRDP(mat);
+#elif AT_USE_URP
+            SetupGlass_URP(mat);
+#else
+            SetupGlass_Standard(mat);
+#endif
+        }
+
+        private static void SetupGlass_HRDP(Material mat) {
+            mat.EnableKeyword(MeshSyncConstants._ENABLE_FOG_ON_TRANSPARENT);
+
+            mat.renderQueue = (int)RenderQueue.Transparent;
+
+            mat.SetFloat(MeshSyncConstants._SurfaceType, 1);
+            mat.SetInt(MeshSyncConstants._ZWrite, 0);
+            mat.SetInt(MeshSyncConstants._SrcBlend, (int)UnityEngine.Rendering.BlendMode.One);
+            mat.SetInt(MeshSyncConstants._DstBlend, (int)UnityEngine.Rendering.BlendMode.OneMinusSrcAlpha);
+            mat.SetFloat(MeshSyncConstants._AlphaSrcBlend, (int)UnityEngine.Rendering.BlendMode.One);
+            mat.SetFloat(MeshSyncConstants._AlphaDstBlend, (int)UnityEngine.Rendering.BlendMode.OneMinusSrcAlpha);
+            mat.SetFloat(MeshSyncConstants._ZTestDepthEqualForOpaque, 4);
+        }
+
+        private static void SetupGlass_URP(Material mat) {
+            mat.SetFloat(MeshSyncConstants._Blend, 1);
+            mat.DisableKeyword(MeshSyncConstants._ALPHATEST_ON);
+            mat.DisableKeyword(MeshSyncConstants._ALPHABLEND_ON);
+            mat.EnableKeyword(MeshSyncConstants._ALPHAPREMULTIPLY_ON);
+            mat.SetInt(MeshSyncConstants._SrcBlend, (int)UnityEngine.Rendering.BlendMode.One);
+            mat.SetInt(MeshSyncConstants._DstBlend, (int)UnityEngine.Rendering.BlendMode.OneMinusSrcAlpha);
+        }
+
+        private static void SetupGlass_Standard(Material mat) {
+            mat.SetFloat(MeshSyncConstants._Mode, 3);
```

Comment:
I guess this is transparent mode ? Please put a comment what this mode means.

Reply:
done